test/distribution.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-env mocha */
+
+require('should')
```

Comment:
true, but `grunt test` was complaining about not using it, as it didn't recognize the `<object>.should` uses